documentation/api/io.strimzi.api.kafka.model.kafka.KafkaClusterSpec.adoc

```diff
@@ -154,18 +150,19 @@ spec:
     logging:
       type: inline
       loggers:
-        kafka.root.logger.level: INFO
-        log4j.logger.kafka.coordinator.transaction: TRACE
-        log4j.logger.kafka.log.LogCleanerManager: DEBUG
-        log4j.logger.kafka.request.logger: DEBUG
-        log4j.logger.io.strimzi.kafka.oauth: DEBUG
-        log4j.logger.org.openpolicyagents.kafka.OpaAuthorizer: DEBUG
+        rootLogger.level: INFO
+        logger.transaction.name: logger.kafka.coordinator.transaction
+        logger.transaction.level: TRACE
+        logger.cleaner.name: logger.kafka.log.LogCleanerManager
+        logger.cleaner.level: DEBUG
+        logger.request.name: logger.kafka.request.logger
+        logger.request.level: DEBUG
+        logger.oauth.name: logger.io.strimzi.kafka.oauth
+        logger.oauth.level: DEBUG
```

Comment:
The names should not the loggers. We should also use the predefined loggers here where possible.

```suggestion
        rootLogger.level: INFO
        logger.logcleaner.level: DEBUG
        logger.requestlogger.level: DEBUG
        logger.transaction.name: kafka.coordinator.transaction
        logger.transaction.level: TRACE
        logger.oauth.name: io.strimzi.kafka.oauth
        logger.oauth.level: DEBUG
```